qemu/tests/cfg/migrate_with_remote.cfg

```diff
@@ -0,0 +1,27 @@
+# Network storage backends:
+#   iscsi_direct
+#   ceph
+# The following testing scenarios are covered:
+#   migrate VM on the same host
+#   migrate VM with postcopy on the same host
+
+- migrate_with_remote: install setup image_copy unattended_install.cdrom
+    only iscsi_direct ceph
+    virt_test_type = qemu
+    type = migration
+    migration_test_command = help
+    migration_bg_command = "cd /tmp; nohup tcpdump -q -i any -t ip host localhost"
+    migration_bg_check_command = pgrep tcpdump
+    migration_bg_kill_command = pkill -9 tcpdump
+    kill_vm_on_error = yes
```

Comment:
`kill_vm_on_error` might be not suppored, you could use `kill_vm = yes`

Reply:
Searched 'kill_vm_on_error' from migration.py as well as py files under virttest, never found it.
Replaced it with kill_vm.